src/test/java/eu/xenit/alfred/content/gateway/security/OidcAuthenticationIntegrationTest.java

```diff
@@ -87,14 +101,14 @@ public void keycloakOIDC_redirectFlow() {
         var initialRedirect = initialResponse.getResponseHeaders().getLocation();
 
         var redirectToKeycloakResponse = rest.get()
-                .uri(Objects.requireNonNull(baseUri.resolve(initialRedirect)))
+                .uri(Objects.requireNonNull(URI.create("http://localhost:" + port).resolve(initialRedirect)))
```

Comment:
Wasn't it better to use a variable here instead of inlining the value everywhere?
Instead of `baseUri`, you can call it `gatewayBaseUri`

Reply:
Ack, resolved